packages/dataclass-editor/src/master/DataClassMasterContent.tsx

```diff
@@ -171,6 +162,16 @@ export const DataClassMasterContent = () => {
         disabled={table.getSelectedRowModel().rows.length === 0}
         aria-label='Delete field'
       />
+      {!context.file.includes('/neo') && (
```

Comment:
Now that it's always visible, I especially vote for having a separator between the buttons.

Reply:
Somthing like this?
![grafik](https://github.com/user-attachments/assets/4765db3f-0c9e-4de6-917b-e9a0d8ac54dc)

I feel like there might be a few too many separators then. If there’s already a separator between each icon, why would we even need them at all? But we can also take a quick look at this together in the office (maybe with Anais) if you’d like.